_posts/2016-06-24_mikemunsell_intro.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title:  "Introducting DSSG Fellow Mike Munsell"
+date:   2016-06-24
+author: Mike Munsell
+---
+
+Hello! My name is Mike Munsell and I am a PhD candidate in Economics at Brandeis University (Boston, MA).  I am a Michigan native and completed my undergraduate degree at the University of Michigan (Go Blue!).  My research interests include the use of machine learning techniques to improve the accuracy of macroeconomic forecasts.
+
+### Why DSSG?
+As an undergraduate, I felt the drive to pursue a career that contributed to the greater good of society.  Equipped with limited knowledge of the types of career paths that were available to me, I did what many undecided freshman do - became pre-med.  However, along the way I discovered how much I enjoyed the study of economics, and also realized that the sight of blood made me faint instantly.  With a new (less nausea-inducing) area to focus my intellectual curiosity, I have intentionally explored ways to apply economics to advancing social welfare throughout my career. 
+
+It is a very exciting time to be a researcher in my field, particularly given the unlocked potential of large datasets in advancing the science of econometric forecasting and policy decision-making.  DSSG will provide me with exposure to solving real-world problems with tools that are directly applicable to economic theory and computation.  I hope to take the skills that I learn this summer and apply it to my own research and collaborative work when I return to Boston in the fall.
+
+### What Do I Want to Learn?
+I'd like to brush up on my programming skills (particularly since I spent the last few months only studying theory to pass my field exams!), as well as expand my knowledge of machine learning techniques.  Additionally, I hope that my time spent in the DSSG program will help me develop research questions that have both societal and data-intensive implications.
+
+### What Have I Learned So Far
```

Comment:
One more tiny nit to pick: I am not an English native speaker, but maybe this should be "What have I learned so far?" or "What I have learned so far"?